src/components/ClaimableBalances.tsx

```diff
@@ -73,13 +96,56 @@ export const ClaimableBalances = () => {
                 </TextLink>
               </td>
               <td>{formatAmount(cb.amount)}</td>
+              <td>
+                <div className="ClaimBalance__buttons">
+                  <Button
```

Comment:
`Button` component is used in “call-to-action” type of use cases. We should use `TextLink` component instead. When `onClick` property is added to this component, it will be treated as a button. You can learn more about this component and our Design System [here](https://design-system.stellar.org/component/textLinks) and [this](https://github.com/stellar/stellar-design-system/blob/main/%40stellar/design-system/src/components/TextLink/index.tsx) repo.

Wrapping `div` might not be necessary.

Reply:
I removed the div, it was indeed not necessary. 

I did not notice the design system before, thank you for point it out.

I don't  understand why we use a button to send a payment transaction but use  TextLink for claiming a balance. I feel they are both call-to-action.

Reply:
You're right, they both are buttons in this case. It's more about the UX. The `Button` component is large, so it gets more attention. If we use `Button` in the data table, it draws a lot of attention (send and receive buttons kind of disappear then). Hope that helps to explain the reasoning behind it. :slightly_smiling_face: Keep asking questions like that!